cross_list_courses/views.py

```diff
@@ -4,23 +4,21 @@
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render
 from django.views.decorators.http import require_http_methods
+
 from lti_permissions.decorators import lti_permission_required
-from django_auth_lti import const
-from django_auth_lti.decorators import lti_role_required
 
 logger = logging.getLogger(__name__)
+PERMISSIONS = settings.TOOL_PERMISSIONS
 
 
 @login_required
-@lti_role_required(const.ADMINISTRATOR)
-@lti_permission_required(settings.PERMISSION_XLIST_TOOL)
+@lti_permission_required(PERMISSIONS['cross_list_courses'])
```

Comment:
reminder to ourselves to validate if this  value  will be 'cross_list_courses' or 'cross_listing' for backward compatibility